docs/features/pixi_global_manifest.md

```diff
@@ -0,0 +1,143 @@
+# Pixi Global Manifest
+
+### Motivation
+
+`pixi global` is currently limited to imperatively managing CLI packages.
+The next iteration of this feature should fulfill the following needs:
+
+- Shareable global environments.
+- Allow managing complex environments with multiple packages as dependencies
+- Flexible exposure of binaries
+
+## Design Considerations
+
+There are a few things we wanted to keep in mind in the design:
+
+1. **User-friendliness**: Pixi is a user focussed tool that goes beyond developers. The feature should have good error reporting and helpful documentation from the start.
+2. **Keep it simple**: The CLI should be all you need to interact with the global environments.
+3. **Unsurprising**: Simple commands should still behave similar to traditional package managers.
+4. **Human Readable**: Any file created by this feature should be human-readable and modifiable.
+
+## Manifest
+
+```toml title="pixi-global.toml"
+# `python_3_10` is the name of the environment
+[envs.python_3_10.dependencies.python]
+spec = "3.10.*"
+expose_binaries = {"python3.10"="python"} # `python` from `python_3_10` will be available as `python3.10`
+
+[envs.python.dependencies.python]
+spec = "3.11.*"
+expose_binaries = "auto" # This will expose python, python3 and python3.11
+
+[envs.python.dependencies.pip]
+spec = "*"
+```
+
+## CLI
+
+Install one or more packages `PACKAGE` into their own environments and expose their binaries
+
+```
+pixi global install [PACKAGE]...
+```
+
+Remove environments of the named packages `PACKAGE`
+```
+pixi global uninstall [PACKAGE]...
+```
+
+Resolve environment without specs
+```
+pixi global upgrade python
+```
+
+Add package `PACKAGE` to environment `ENV`.
+If environment `ENV` does not yet exist, then it will be created.
+```
+pixi global add --name ENV PACKAGE
```

Comment:
Does `add` auto expose binaries or not? and if so can I expose them from this command?

Reply:
It does not expose binaries. Adding an option could be discussed, but my gut feeling is that it's better to not do that.

Reply:
maybe we expose binaries like `install` does also for `add`? 
and add an option not to expose binaries?

my rationale is that if I do:

`pixi global install jupyterlab` and then
`pixi global add pip ( or another binary tool ) --name jupyterlab`

pip will not be available in jupyterlab because we do not expose binaries

Reply:
If we went for that, `pixi global add` and `pixi global install` would behave exactly the same if `--expose` is given, but differently if it is not.

Could still work, just wanted to mention it here

Reply:
```
> pixi global install python
> pixi global inject pytest -e python

> pytest
command not found

> pixi global expose-binary -e python pytest:pytest
> pytest --version
x.y.z

> pixi global install python=3.9 -e python39
ERROR: python is already available as a binary from `python` global installation
please use `--expose python39:python`
> pixi global install python=3.9 -e python39 --expose python39:python
> python39
>>> exit()
> 
```
After a discussion we came to this workflow